modal/cls.py

```diff
@@ -397,6 +397,22 @@ def _get_partial_functions(self) -> dict[str, _PartialFunction]:
             raise AttributeError("You can only get the partial functions of a local Cls instance")
         return _find_partial_methods_for_user_cls(self._user_cls, _PartialFunctionFlags.all())
 
+    def _get_app(self) -> "modal.app._App":
```

Comment:
Why do we need these new private getters?

Reply:
To allow access to the variable from the synchronicity wrapper without ugly `synchronizer._translate_in(...)` calls 😬 
Maybe synchronicity should simply getattr/setattr everything to the underlying instance (not sure why we aren't doing that), but that feels like a larger change